about/async/suggest.js

```diff
@@ -53,11 +54,50 @@ exports.create = function (api) {
       suggestion,
       {idle: true}
     )
-    suggestions = map(contacts, suggestion, {idle: true})
+
+    const suggestionsRecord = computed(contacts, contacts => {
+      var result = {}
+      contacts.forEach(contact => {
+        result[contact] = api.about.obs.names(contact)
+      })
+
+      return result
+    })
```

Comment:
I would probably use a `mutant/lookup` here. Slightly nicer code and it would be a little more efficient as it wouldn't have to recompute the whole result every time.

```
const suggestionsRecord = lookup(contacts, contact => {
  return [contact, api.about.obs.names(contact)]
})
```